content/en/serverless/custom_metrics/_index.md

```diff
@@ -3,56 +3,95 @@ title: Custom Metrics
 kind: documentation
 ---
 
-If your Lambda functions are sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code. If you want to send new data not captured in an existing log or trace, you can [send custom metrics using the Datadog Lambda Library](#sending-custom-metrics-from-the-datadog-lambda-library).
+## Overview
+
+There are a few different ways to submit custom metrics to Datadog from a Lambda function. 
+
+- **Creating custom metrics from logs or traces**: If your Lambda functions are already sending trace or log data to Datadog, and the data you want to query is captured in an existing log or trace, you can [generate custom metrics from logs and traces](#creating-custom-metrics-from-logs-or-traces) without re-deploying or making any changes to your application code.
+- **Submitting custom metrics synchronously**: If you want to generate custom metrics from a non-performance-critical Lambda function written in Python, Node.js or Golang and you are not interested in collecting traces, you can use Datadog Lambda Library to [submit custom metrics synchronously](#enabling-synchronous-custom-metrics).
+- **Submitting custom metrics asynchronously**: If you want to generate custom metrics from a performance-critical Lambda function, Datadog encourage you to use Datadog Lambda Library to [submit custom metrics asynchronously](#enabling-asynchronous-custom-metrics).
+- **(Deprecated) Submitting custom metrics from CloudWatch logs**: The method to submit custom metrics by printing a log formatted as `MONITORING|<UNIX_EPOCH_TIMESTAMP>|<METRIC_VALUE>|<METRIC_TYPE>|<METRIC_NAME>|#<TAG_LIST>` has been [deprecated](#deprecated-cloudwatch-logs), and you should migrate to one of the solutions above.
+- **(Not recommended) Using a third-party library**: Most of [third-party libraries](#third-party-libraries) do not submit metrics as distributions and can lead to under-counted results.
+
+### Understanding distribution metrics
+
+Custom metrics submitted from Lambda functions are aggregated as [distributions][1], as they are designed to instrument applications, independent of the underlying hosts. You can query the metrics using aggregation `avg`, `sum`, `max`, `min`, `count`. You can also enable percentile aggregations (p50, p75, p90, p95, p99) and [manage tags][2] for aggregation on the metric summary page.
+
+Certain submission methods allow you submit metrics with timestamps from the past, up-to 20 mins old.
```

Comment:
When I read this, I immediately ask "which methods?". I think the answer is anything except the Extension. Since support for this is inconsistent, do we want to encourage it at all?

Reply:
Good feedback, let me call it out more explicitly.